cmake/modules/AddSwift.cmake

```diff
@@ -1592,6 +1602,12 @@ function(add_swift_library name)
          endif()
        endif()
 
+       if(NOT "${sdk}" STREQUAL "WINDOWS" AND NOT "${name}" STREQUAL "swiftRemoteMirror")
+          # Add back the flags we removed from the call to HandleLLVMOptions 
+          # in SwiftSharedCMakeConfig.cmake.
+          list(APPEND swiftlib_link_flags_all "-Wl,-z,defs")
```

Comment:
I am assuming this goes away after c++14?

Reply:
This part doesn't. The flag is currently being unconditionally removed so Windows works, (https://github.com/apple/swift/pull/13209), so this adds it back for non-Windows targets. swiftRemoteMirror was also removing the flag manually, so I consolidated that change here.